### ISSUE TITLE
Fix description

### ISSUE THREAD

**Issue by MasterOfTouch** [int02h/manifest-guard#15]
![image](https://user-images.githubusercontent.com/37360486/212289311-ef2c4abd-15f3-4e83-b1b2-23687c50651e.png)
Also would be nice to mention that `manifestGuard {}` must be set in build.gradle of app module or show the usage of script in sample.

**Comment by int02h** [comment]
Hello!
Thank your request. README file was fixed.

> Also would be nice to mention that manifestGuard {} must be set in build.gradle of app module

That is said in the first sentence in Setup section: "Also would be nice to mention that manifestGuard {} must be set in build.gradle of app module". Do you think this sentence should be rephrased?

> show the usage of script in sample.

Yes, that's a good point. Currently, the sample relies only on the default config but meantime I didn't sort out how to show all possible configs in one sample.

**Comment by MasterOfTouch** [comment]
may be its only my lack of expierence of working with gradle plugins. but i had to spent some time to realize of where to insert manifestGuard {}

**Comment by MasterOfTouch** [comment]
also
 ```
maven {
            url "https://plugins.gradle.org/m2/"
        }
```
confusing since it's literally gradlePluginPortal()
https://github.com/gradle/gradle/pull/613

**Comment by int02h** [comment]
Yes, I usually copy code samples from [official page](https://plugins.gradle.org/plugin/com.dpforge.manifestguard).

**Comment by int02h** [comment]
Almost always blocks for configuring plugins are placed just on the top level of build.gradle
